scss/_custom-styles.scss

```
@@ -254,3 +254,11 @@
   // stylelint-disable-next-line declaration-no-important
   border-width: .25rem !important;
 }
+
+// >> Figure caption.
+
+.figure-caption {
+  padding-top: .5em;
+  padding-bottom: .9375em;
+  border-bottom: 1px solid #eef1f1;
```

Alternate color suggestion.
```suggestion
  border-bottom: 1px solid $hr-border-color;
```